packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -372,6 +379,10 @@ contract EpochRewards is Ownable, Initializable, UsingPrecompiles, UsingRegistry
    * @return The per validator epoch payment and the total rewards to voters.
    */
   function calculateTargetEpochPaymentAndRewards() external view returns (uint256, uint256) {
+    if (frozen) {
```

Comment:
nit: This *shouldn't* strictly be necessary, since if one epoch rewards action fails we won't execute any of them.

Reply:
Yep. @nategraf recommended this change for robustness.

One nice thing about how this is implemented is that after Baklava we can just delete the `baklava/` directory and then remove any code that the compiler complains about.